src/Column/ColumnFactory.php

```diff
@@ -104,6 +104,12 @@ public function fromType(string $type, array $info = []): ColumnInterface
         return $column;
     }
 
+    public function fromPseudoType(string $pseudoType, array $info = []): ColumnInterface
+    {
+        // PostgreSQL doesn't support unsigned types
+        return parent::fromPseudoType($pseudoType, $info)->unsigned(false);
```

Comment:
I do not think that this is correct. There are ways how to get unsigned type (with check)